R/filter_loss.R

```diff
@@ -4,7 +4,17 @@
 ##
 
 
-## recursive filter with a constant loss
+#' Recursive filter with a constant loss
+#'
+#' Description
+#'
+#' @param x Placeholder
+#' @param a Placeholder
+#' @param loss Placeholder
+#' @param init Placeholder
+#'
+#' @useDynLib hydromad filter_constloss
+#' @export
 filter_loss <-
```

Comment:
Was this exported originally? If not, why did it need to be?
I'm on the fence as to whether it should be or not

Reply:
It was exported in the original NAMESPACE but it read:

`export(filter_loss) # export?`

I wasn't sure what you wanted with it, so I kept the NAMESPACE the same for the time being.

Reply:
Addressed by #69. Thanks!